tapiriik/services/service.py

```diff
@@ -25,13 +25,14 @@ def FromID(id):
         raise ValueError
 
     def List():
-        return [RunKeeper, Strava, GarminConnect, SportTracks, Dropbox, TrainingPeaks, RideWithGPS, TrainAsONE, Endomondo, Motivato, NikePlus, VeloHero, TrainerRoad, Smashrun, BeginnerTriathlete] + PRIVATE_SERVICES
+        return [Pulsstory, RunKeeper, Strava, GarminConnect, SportTracks, Dropbox, TrainingPeaks, RideWithGPS, TrainAsONE, Endomondo, Motivato, NikePlus, VeloHero, TrainerRoad, Smashrun, BeginnerTriathlete] + PRIVATE_SERVICES
 
     def PreferredDownloadPriorityList():
         # Ideally, we'd make an informed decision based on whatever features the activity had
         # ...but that would require either a) downloading it from evry service or b) storing a lot more activity metadata
         # So, I think this will do for now
         return [
+		    Pulsstory,
```

Comment:
Given that the implementation is mostly copied from RunKeeper, and seems to support the same activity data components, I'd expect to see this further down the priority list with RK.